controllers/hetznercluster_controller_test.go

```diff
@@ -848,90 +848,102 @@ func TestSetControlPlaneEndpoint(t *testing.T) {
 		hetznerCluster := &infrav1.HetznerCluster{Spec: infrav1.HetznerClusterSpec{ControlPlaneLoadBalancer: infrav1.LoadBalancerSpec{Enabled: false}, ControlPlaneEndpoint: nil}}
 		got := SetControlPlaneEndpoint(hetznerCluster)
 		if (hetznerCluster.Spec.ControlPlaneLoadBalancer.Enabled == true){
-			t.Fatal("Wrong value for ControlPlaneLoadBalancer.Enabled")
+			t.Fatalf("Wrong value for ControlPlaneLoadBalancer.Enabled. Got: %v, Want: false", hetznerCluster.Spec.ControlPlaneLoadBalancer.Enabled)
 		}
 		if (hetznerCluster.Spec.ControlPlaneEndpoint != nil){
-			t.Fatal("Wrong value for hetznerCluster.Spec.ControlPlaneEndpoint")
+			t.Fatalf("Wrong value for hetznerCluster.Spec.ControlPlaneEndpoint. Got: %s, Want: '<nil>'", hetznerCluster.Spec.ControlPlaneEndpoint)
```

Comment:
Want: nil. In this case we actually want nil, not the string "<nil>" that we have as well in the code